programs/util.h

```diff
@@ -118,7 +120,8 @@ U64 UTIL_getFileSize(const char* infilename);
 U64 UTIL_getTotalFileSize(const char* const * fileNamesTable, unsigned nbFiles);
 int UTIL_getFileStat(const char* infilename, stat_t* statbuf);
 int UTIL_setFileStat(const char* filename, stat_t* statbuf);
-int UTIL_chmod(char const* filename, mode_t permissions);   /*< like chmod, but avoid changing permission of /dev/null */
+int UTIL_chmod(char const* filename, mode_t permissions, int onlyRegularFiles);   /*< like chmod, but avoid changing permission of special files */
```

Comment:
Suggestion : if we agree that `chmod()` should only be applied to regular files, then it's not even necessary to provide a parameter `onlyRegularFiles`. It's implied as always `1`.

Same thing for `chown()` below.